chainnotifier_client.go

```diff
@@ -170,18 +204,27 @@ func (s *chainNotifierClient) RegisterConfirmationsNtfn(ctx context.Context,
 					errChan <- err
 					return
 				}
+
+				block, err := decodeBlock(c.Conf.RawBlock)
```

Comment:
If `includeBlock=false`, wouldn't `c.Conf.RawBlock` be empty / fail deserialization in `decodeBlock`?

Reply:
Yes, I think that should be guarded by another `if` or return early in `decodeBlock`.